source/_integrations/microsoft_graph.markdown

```diff
@@ -0,0 +1,76 @@
+---
+title: Microsoft Graph
+description: Instructions on how to query Microsoft's Graph API in Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.119
+ha_codeowners:
+  - '@jlweston85'
+ha_domain: microsoft_graph
+ha_config_flow: true
+---
+
+The Microsoft Graph integration allows Home Assistant to create sensors from various properties of your Office 365 tenant via Microsoft's Graph API. All queries are performed using the authenticated user's account and depend on the permissions granted when creating the Azure AD application.
+
+Home Assistant authenticates with Microsoft through OAuth2. Set up your credentials by completing the steps in [Manual Configuration](#manual-configuration) and then add the integration in **Configuration -> Integrations -> Microsoft Graph**. Ensure you log in using a Microsoft account that belongs to your organization's tenant.
```

Comment:
This is not part of the introduction, these are configuration/set up instructions.

Reply:
I'll delete the second paragraph if it doesn't belong here. Once it's clearer that all of the configuration described is mandatory, this paragraph adds no value anyway.